tasks/bash.sh

```diff
@@ -1,106 +1,153 @@
-#!/usr/bin/env bash
-# The install_shell.sh implementation from the puppetlabs-puppet_agent module can use this
-# by passing positional argument with value "platform" or "release" to get only the platform
-# or version string only. 
-
-minor () {
-    minor="${*#*.}"
-    [ "$minor" == "$*" ] || echo "${minor%%.*}"
+#!/bin/bash
```

Comment:
Slowly standardizing on this shebang for bash? looks like we have done it for package and service. Next up would be puppet_agent::install. I guess now that bolt allows configuration of interpreters its not as big a deal.